test/pubsub.ts

```diff
@@ -243,6 +251,15 @@ describe('PubSub', () => {
       assert.deepStrictEqual(pubsub.api, {});
     });
 
+    it('should default to the opened state', () => {
+      assert.strictEqual(pubsub.isOpen, true);
+    });
+
+    it('should not be in the opened state after close()', async () => {
```

Comment:
might be tempted to flesh this test out a bit more, checking that the internal state reflects that things were closed, as an example has the gRPC stream actually had close called on it? has `closeAllClients_` resovled.

Reply:
This test does look really minimal, but I think those points are covered elsewhere, actually. e.g. this shouldn't resolve until `closeAllClients_` resolves:
```
      await pubsub.close();
```

There's a separate test that stubs out `close` for the gax clients in `closeAllClients_`, so that should also be covered.